apps/src/StudioApp.js

```diff
@@ -29,6 +29,7 @@ var FeedbackUtils = require('./feedback');
 var VersionHistory = require('./templates/VersionHistory.jsx');
 var Alert = require('./templates/alert.jsx');
 var codegen = require('./codegen');
+var PuzzleRatingUtils = require('./puzzleRatingUtils');
```

Comment:
convention is to use lower case here (puzzleRatingsUtils). Upper case implies instantiable class.